lighthouse-core/runner.js

```diff
@@ -21,7 +21,7 @@ const Sentry = require('./lib/sentry');
 class Runner {
   static run(connection, opts) {
     // Clean opts input.
-    opts.flags = opts.flags || {};
```

Comment:
you can also remove flags from ` return Runner.run(connection, {url, flags, config})` in index.js